docs/pages/aws-oss-guide.mdx

```diff
@@ -675,5 +675,3 @@ below to Teleport Nodes in `etc/teleport.yaml` to have helpful labels in the Tel
         command: [curl, "http://169.254.169.254/latest/meta-data/public-ipv4"]
         period: 1h0m0s
 ```
-
-Create labels based on [EC2 Tags](https://community.gravitational.com/t/setting-teleport-labels-based-on-ec2-tags-for-use-with-rbac/558).
```

Comment:
I've ported this guide here:

https://goteleport.com/docs/setup/guides/ec2-tags/

Reply:
I'll fix it in this PR -- but be aware I copied this part of the diff from [86a6ce7e9f870298a1970c9c417011afde75325d](https://github.com/gravitational/teleport/commit/86a6ce7e9f870298a1970c9c417011afde75325d#diff-1108c93a3b2725cf4a40894fe6e99a29cd289c1fc2d895d058449ac9952fe54eL680) -- so other branches probably lost this too.

Reply:
I linked to the main version in 2d6a684e69b38c38fe183ef58e473e381f3dd212.

This is a bit fragile  -- it will inevitably go bad since it is linking across doc versions. I chose this because I'm pretty burnt out on chasing docs stuff tonight and don't think there is high ROI in me backporting b2e67fd. :shrug: Let me know what you think.